dashboard/test/models/script_test.rb

```diff
@@ -1602,26 +1602,42 @@ class SummarizeVisibleAfterScriptTests < ActiveSupport::TestCase
     script_copy = script.clone_with_suffix('copy')
     assert_equal 'test-fixture-variants-copy', script_copy.name
 
-    assert_equal 1, script_copy.script_levels.count
-    sl = script_copy.script_levels.first
+    assert_equal 4, script_copy.script_levels.count
 
+    sl = script_copy.script_levels[0]
```

Comment:
Optional - these asserts could be made a lot shorter if you refactored it into two loops, one for all the asserts aside from name, and one for the name. Or it could be one loop and you could use Array.zip to zip `script_copy.script_levels` with the list of expected names. YMMV on whether any of those are better, though.

Reply:
I've implemented the first option. thanks for this tip!